openedx/core/djangolib/net_utils.py

```diff
@@ -0,0 +1,24 @@
+"""
+Helper functions and classes for network utilities
+"""
+
+
+from ipware.ip2 import get_client_ip
+
+
+def get_ip_address(request, real_ip_only=False):
+    """
+    Acquires the client's IP Adress using django-ipware
+    """
+    if (real_ip_only):
+        proxy_order="right-most"
+    else:
+        proxy_order="left-most"
+
+    client_ip, is_routable = get_client_ip(request, proxy_order)
+
+    if (real_ip_only and (not is_routable)):
```

Comment:
Change `if (real_ip_only and (not is_routable))` -> `if real_ip_only and (not is_routable)`